app/grandchallenge/core/management/commands/archive_item_permissions.py

```diff
@@ -0,0 +1,18 @@
+from django.core.management import BaseCommand
+from django.core.paginator import Paginator
+
+from grandchallenge.archives.models import ArchiveItem
+
+
+class Command(BaseCommand):
+    def handle(self, *args, **options):
+        items = (
+            ArchiveItem.objects.select_related("archive")
+            .order_by("-created")

```

Comment:
Needs to be oldest first otherwise we get problems with insertions when running this:

```suggestion
            .order_by("created")
```